tests/test_playbooks/host_info.yml

```diff
@@ -54,7 +54,7 @@
     - name: check host details
       assert:
         that:
-          - host_info['host']['name'] == "test-host.{{ host.domain }}"
+          - host_info['host']['name'] == "test-host.{}".format(host.domain)
```

Comment:
As if that were easier...
So it looks like the whole condition is put through jinja2. Is that correct?

Reply:
OTOH, you can use `|filter`, so maybe there is still some jinja... the hell I know

Reply:
That's what it looks to me. Whatever you assert here seems to be processed by jinja.

Reply:
instead of melting my brain over this, I'll go get some sushi :)